packages/core/admin/admin/src/StrapiApp.js

```diff
@@ -90,11 +129,31 @@ class StrapiApp {
       const boot = this.appPlugins[plugin].boot;
 
       if (boot) {
-        boot({ getPlugin: this.getPlugin });
+        boot({
+          addSettingsLink: this.addSettingsLink,
+          addSettingsLinks: this.addSettingsLinks,
+          getPlugin: this.getPlugin,
+        });
       }
     });
   }
 
+  createSection = (section, links) => {
```

Comment:
Can we be more specific about the name of the function to clearly understand what it does? Something like `createSettingSection` might be more relevant, WDYT? Without knowing the API, I don't know what section will be created.